go/obscuronode/enclave/storage.go

```diff
@@ -70,12 +70,25 @@ type Storage interface {
 	FetchSecret() SharedEnclaveSecret
 	// StoreSecret stores a secret in the enclave
 	StoreSecret(secret SharedEnclaveSecret)
+
+	// StoreRollupGenesis stores the rollup genesis
+	StoreRollupGenesis(rol *Rollup)
```

Comment:
why not: StoreGenesisRollup/FetchGenesisRollup ?

Reply:
Sure 👍